11ty/definitions/mansplaining.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Mansplaining
+slug: mansplaining
+speech: verb
+defined: true
+reading:
+- text: "Rebecca Solnit's 'Men Explain Things to Me' (2008)"
+  href: https://www.guernicamag.com/rebecca-solnit-men-explain-things-to-me/
+- text: "Kim Goodwin's 'Am I mansplaining? graphic"
+  href: https://ichef.bbci.co.uk/wwfeatures/live/1600_900/images/live/p0/6f/zj/p06fzjxg.jpg
+---
+
+A man explaining a topic to a woman in a condescending, overconfident way, without acknowledging the fact that the woman often knows more about that topic than the man doing the explaining. 
+
+Examples include but aren't limited to: a man explaining to a woman how to pronounce her own name, explaining menstruation, or explaining the history of a country with which the woman is associated but the man has never been to.  
+
+## Note
+
+A good way to check if you're mansplaining is to ask yourself whether: 
+
+(1) the person wants the explanation
+
+(2) you are assuming incompetence, and, 
+
+(3) how bias affects your interpretation of (1) and (2)
+
```

Comment:
```suggestion
### Suggestions
To avoid mansplaining, a good way is to:
- **Pause.** It's always a good idea to think before you talk/type.
- **Ask yourself if your voice is really adding a necessary perspective.** Is this a post written for an audience you're not a part of? Are you an outside observer? It might be best to leave it be and learn from it internally.
- **Check the person's credentials.** Are you, someone with armchair knowledge of viruses, about to explain something to an epidemiologist? It's not okay to mansplain to anyone, but it's especially infuriating if the person has tangible credentials that you do not, and you're trying to explain to them.
- **Ask for consent. Do not expect a "yes."** "Would you like my thoughts on this subject? May I add my perspective here as an amateur epidemiologist who read one uncited paper?"
- **Leave room for and presume knowledge, reflecting on your own knowledge** "You might already know about this, but I found it surprising that..." (The critical part of this example is that it turns it back on you; "__I__ found it surprising that..."
-- 
```